src/frontend/src/components/Stats.js

```diff
@@ -1,14 +1,29 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { useAuth0 } from "@auth0/auth0-react";
 import { Link } from "react-router-dom";
+import { getStatsApi } from "./util/apiCalls";
 
 export default function Stats() {
-  // TODO   make statsData to a useState hook calling gameStruct object
-  // Populating dummy data
-  const statsData = {
-    gamesPlayed: 10,
-    winPercentage: 70, // In percentage
-    mostGuessedWord: "hello",
-  };
+  const [statsData, setStatsData] = useState({
+    gamesPlayed: 0,
+    winPercentage: 0,
+    mostGuessedWord: "",
+  });
+  const { isAuthenticated, user } = useAuth0();
+
+  useEffect(() => {
+    // ? not sure if we're only using user or isAuth as well.
+    if (user) {
```

Comment:
Please change this to a more robust check like 
`if(isAuthenticated && user != null)`